bakerydemo/static/css/main.css

```diff
@@ -665,7 +688,7 @@ span.outline {
   padding: 0;
 }
 
-.bread-list-item .image:hover img {
+.bread-list-item:hover img {
```

Comment:
This should be defined on hover on the `a` tags rather than the whole div – otherwise it looks as if the hovered area is clickable even though it's not. Or otherwise make this `bread-list-item` element the `a` tag.

See how the pointer cursor is only on specific parts of the list item:

![mobile-amends-hover](https://user-images.githubusercontent.com/877585/34115247-bd27e3ec-e41d-11e7-8270-e61b017e6955.gif)

Reply:
I'm not even sure why I've amended a hover on mobile styling `:facepalm:`

Will go and look at the hover though because you're right it makes it confusing.

Reply:
TBH I don't think it brings _that much_ to the UI to make it orange-ish like that (it's images of bread...), so this could simply be removed.